src/main/java/io/orkes/conductor/client/http/OrkesWorkflowClient.java

```diff
@@ -19,10 +19,9 @@
 import java.util.UUID;
 import java.util.concurrent.*;
 
-import com.netflix.conductor.common.metadata.workflow.*;
-import io.orkes.conductor.client.model.JumpWorkflowExecutionRequest;
 import org.apache.commons.lang.StringUtils;
 
+import com.netflix.conductor.common.metadata.workflow.*;
```

Comment:
I know isn't your change, but we shouldn't use wildcard imports

Reply:
There is a spotless plugin that mandates this.